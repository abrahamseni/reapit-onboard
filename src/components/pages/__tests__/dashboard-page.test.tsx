import { render } from '@testing-library/react'
import { createBrowserHistory } from 'history'
import React from 'react'
import { Router, Switch } from 'react-router-dom'
import Dashboard from '../dashboard'

const history = createBrowserHistory()

describe('Dashboard', () => {
  it('should match a snapshot', () => {
    expect(
      render(
        <Router history={history}>
          <Switch>
            <Dashboard />
          </Switch>
        </Router>,
      ),
    ).toMatchSnapshot()
  })
})
