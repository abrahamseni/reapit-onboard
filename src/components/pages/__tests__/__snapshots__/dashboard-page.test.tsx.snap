// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard should match a snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <mock-styled-div>
        <mock-styled-aside>
          <mock-styled-h1>
            Marketplace
          </mock-styled-h1>
          <mock-styled-div
            classname="el-mb9"
          >
            <mock-styled-div>
              Home
            </mock-styled-div>
            <mock-styled-div>
              Tennants
            </mock-styled-div>
            <mock-styled-div>
              Agents
            </mock-styled-div>
            <mock-styled-div>
              Messages
            </mock-styled-div>
            <mock-styled-div>
              Invoices
            </mock-styled-div>
            <mock-styled-div>
              Settings
            </mock-styled-div>
          </mock-styled-div>
        </mock-styled-aside>
        <mock-styled-section>
          <mock-styled-header>
            <mock-styled-h2>
              Nina Smith - Super Agent
            </mock-styled-h2>
            <mock-styled-div>
              <mock-styled-img
                alt="avatar"
                src="https://images.unsplash.com/photo-1573600073955-f15b3b6caab7?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=415&q=80"
              />
              <mock-styled-div>
                <svg
                  fill="currentColor"
                  height="18"
                  stroke="currentColor"
                  stroke-width="0"
                  viewBox="0 0 16 16"
                  width="18"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M8 16a2 2 0 0 0 2-2H6a2 2 0 0 0 2 2zM8 1.918l-.797.161A4.002 4.002 0 0 0 4 6c0 .628-.134 2.197-.459 3.742-.16.767-.376 1.566-.663 2.258h10.244c-.287-.692-.502-1.49-.663-2.258C12.134 8.197 12 6.628 12 6a4.002 4.002 0 0 0-3.203-3.92L8 1.917zM14.22 12c.223.447.481.801.78 1H1c.299-.199.557-.553.78-1C2.68 10.2 3 6.88 3 6c0-2.42 1.72-4.44 4.005-4.901a1 1 0 1 1 1.99 0A5.002 5.002 0 0 1 13 6c0 .88.32 4.2 1.22 6z"
                  />
                </svg>
                <mock-styled-span>
                  <mock-styled-p>
                    5
                  </mock-styled-p>
                </mock-styled-span>
              </mock-styled-div>
            </mock-styled-div>
          </mock-styled-header>
        </mock-styled-section>
      </mock-styled-div>
    </div>
  </body>,
  "container": <div>
    <mock-styled-div>
      <mock-styled-aside>
        <mock-styled-h1>
          Marketplace
        </mock-styled-h1>
        <mock-styled-div
          classname="el-mb9"
        >
          <mock-styled-div>
            Home
          </mock-styled-div>
          <mock-styled-div>
            Tennants
          </mock-styled-div>
          <mock-styled-div>
            Agents
          </mock-styled-div>
          <mock-styled-div>
            Messages
          </mock-styled-div>
          <mock-styled-div>
            Invoices
          </mock-styled-div>
          <mock-styled-div>
            Settings
          </mock-styled-div>
        </mock-styled-div>
      </mock-styled-aside>
      <mock-styled-section>
        <mock-styled-header>
          <mock-styled-h2>
            Nina Smith - Super Agent
          </mock-styled-h2>
          <mock-styled-div>
            <mock-styled-img
              alt="avatar"
              src="https://images.unsplash.com/photo-1573600073955-f15b3b6caab7?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=415&q=80"
            />
            <mock-styled-div>
              <svg
                fill="currentColor"
                height="18"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 16 16"
                width="18"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M8 16a2 2 0 0 0 2-2H6a2 2 0 0 0 2 2zM8 1.918l-.797.161A4.002 4.002 0 0 0 4 6c0 .628-.134 2.197-.459 3.742-.16.767-.376 1.566-.663 2.258h10.244c-.287-.692-.502-1.49-.663-2.258C12.134 8.197 12 6.628 12 6a4.002 4.002 0 0 0-3.203-3.92L8 1.917zM14.22 12c.223.447.481.801.78 1H1c.299-.199.557-.553.78-1C2.68 10.2 3 6.88 3 6c0-2.42 1.72-4.44 4.005-4.901a1 1 0 1 1 1.99 0A5.002 5.002 0 0 1 13 6c0 .88.32 4.2 1.22 6z"
                />
              </svg>
              <mock-styled-span>
                <mock-styled-p>
                  5
                </mock-styled-p>
              </mock-styled-span>
            </mock-styled-div>
          </mock-styled-div>
        </mock-styled-header>
      </mock-styled-section>
    </mock-styled-div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
